src/plugins/explore/public/application/legacy/discover/application/components/top_nav/get_top_nav_links.tsx

```diff
@@ -66,7 +67,7 @@ export const getTopNavLinks = (
       });
       // TODO this behavior is different from Discover. Clicking New in Explore
       // only closes the saved search and does not change the query.
-      store.dispatch(setSavedSearch(undefined));
+      store!.dispatch(setSavedSearch(undefined));
```

Comment:
Is this for fixing the type error? I prefer to check if store is not undefined first instead of adding "!" here.